modules/openshift-developer-cli-odo-notable-features.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-release-notes.adoc
+
+[id="odo-notable-features_{context}"]
+= Notable Features in {odo-title}{nbsp}{odo-ver}
+
+{odo-title}{nbsp}{odo-ver} offers a simple way to develop applications on {product-title}. {odo-title} is completely client-based and requires no server within {product-title} cluster.
+
+{odo-title}{nbsp}{odo-ver} comes with:
+
+* Simple syntax and design centered around concepts familiar to developers such as projects, applications and components.
+* Compatibility with any language or runtime within the OpenShift catalog of component types.
+* Capability to add custom component types - custom image builders.
```

Comment:
Suggest maybe "using" instead of the dash here? 

`Capability to add custom component types using custom image builders.`